### PR TITLE
added html-css

### LINKED ISSUE #1: HTML/CSS
To create an HTML and CSS skeleton on a master branch.

### PR DESCRIPTION
**Pull request by gelilaa**
closes #1

### CODE REVIEW

**Review by fmkarakus — APPROVED** [review]
Very well done and tasteful :rocket:
I think it would be better if we add a form for add/modify questions.

**Comment by gelilaa** [comment]
@fmkarakus I was thinking the same thing also. So I'm half way with the add form. I will send you a pic on Slack

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset='utf8'>
```
Comment (abelRoland) [inline index.html:5]: this part of the template always give error, the correct is `<meta chartset='utf-8'>`

**Review by abelRoland — CHANGES_REQUESTED** [review]
Great job. Thanks to share!

File: index.html (hunk 1 above)
Comment (gelilaa) [reply index.html:5]: I always get tricked by it. 😄  Thank you for pointing it out @abelRoland . I will clean it up with the second batch of PR!

**Review by yauheniya-askolkava — APPROVED** [review]
good for me, thank you!

**Review by abelRoland — COMMENTED** [review]
Great!

**Pull request merged**